ecs.php

```diff
@@ -124,405 +128,417 @@
-        $services->set(PhpdocReturnSelfReferenceFixer::class);
-        // Scalar types should always be written in the same form.
-        $services->set(PhpdocScalarFixer::class);
-        // Single line `@var` PHPDoc should have proper spacing.
-        $services->set(PhpdocSingleLineVarSpacingFixer::class);
-        // PHPDoc should start and end with content
-        $services->set(PhpdocTrimFixer::class);
-        // The correct case must be used for standard PHP types in PHPDoc.
-        $services->set(PhpdocTypesFixer::class);
-        // `@var` and `@type` annotations must have type and name in the correct order
-        $services->set(PhpdocVarAnnotationCorrectOrderFixer::class);
-        // PHP code must use the long `<?php` tags or short-echo `<?=` tags and not other tag variations
-        $services->set(FullOpeningTagFixer::class);
-        // PHPUnit assertion method calls like ->assertSame(true, $foo) should be written with dedicated method like ->assertTrue($foo)
-        $services->set(PhpUnitConstructFixer::class);
-        // PHPUnit assertions like assertInternalType, assertFileExists, should be used over assertTrue
-        $services->set(PhpUnitDedicateAssertFixer::class);
-        // PHPUnit assertions like assertIsArray should be used over assertInternalType
-        $services->set(PhpUnitDedicateAssertInternalTypeFixer::class);
-        // Use dedicated helper methods createMock() and createPartialMock() where possible
-        $services->set(PhpUnitMockFixer::class);
-        // Use of eg. ->will($this->returnValue(..)) must be replaced by its shorter equivalent such as ->willReturn(...)
-        $services->set(PhpUnitMockShortWillReturnFixer::class);
-        // Use expectedException*() methods instead of @expectedException* annotation (both following fixers must be applied to do so)
-        $services->set(PhpUnitNoExpectationAnnotationFixer::class);
+            // Probably forgotten debug calls, use logger instead
+            'dump' => null,
+            'echo' => null,
+            'phpinfo' => null,
+            'print_r' => null,
+            'printf' => null,
+            'var_export' => null,
+            'var_dump' => null,
+        ],
+    ])
+    // There should be exactly one blank line before a namespace declaration.
+    ->withConfiguredRule(BlankLinesBeforeNamespaceFixer::class, ['min_line_breaks' => 2, 'max_line_breaks' => 2])
+    // Proper operator spacing
+    ->withConfiguredRule(OperatorSpacingSniff::class, ['ignoreNewlines' => true])
+    // PHP arrays should be declared using the configured syntax
+    ->withConfiguredRule(ArraySyntaxFixer::class, ['syntax' => 'short'])
+    // The body of each structure MUST be enclosed by braces. Braces should be properly placed
+    ->withConfiguredRule(BracesFixer::class, ['allow_single_line_closure' => true, 'allow_single_line_anonymous_class_with_empty_body' => true])
```

Comment:
Deprecated fixer, see https://cs.symfony.com/doc/rules/basic/braces.html

Also the line is over 120 chars :)

Reply:
Removed